check/data

```diff
@@ -1,160 +0,0 @@
----
```

Comment:
See comment elsewhere, I want to keep the tests that use this data

Reply:
See previous comment on test files.

Reply:
There are some new tests in bench_common that might cover at least some of this, but I suspect we are losing some useful tests here?